README.md

```diff
@@ -35,15 +35,28 @@ packages/example/example.tgz
 packages/example/example.conf
 ```
 
-If you want a `zip` rather than a `tgz` built, instead add
+If you want a `zip` rather than a `tgz` built, instead add the following to your `build.sbt`.
+
 ```
 import com.typesafe.sbt.packager.Keys._
+```
+for sbt-native-packager 0.8.0, OR
+```
+enablePlugins(SbtNativePackager)
```

Comment:
I think we can write this as `enablePlugins(UniversalPlugin)` without anything else for a basic Java app, or say `enablePlugins(JDebPackaging, JavaServerAppPackaging)` if we want support for running it as a daemon.